packages/frontend/app/components/app/sidebar/actions/action/index.hbs

```diff
@@ -0,0 +1,8 @@
+<div data-test-action-row class='sidebar__actions__action'>
+
+  <span data-test-action-name>
+    {{or @action.header '[error]'}}
```

Comment:
what is '[error]'? when would it be displayed?

Reply:
It was added as a placeholder for if actions had an issue with displaying the action name. Debating whether to turn it into an actual error message or removing it, as actions without headers should theoretically be caught on addition.